rdoc.gemspec

```diff
@@ -1,6 +1,4 @@
-# -*- encoding: utf-8 -*-
-$:.unshift File.expand_path("../lib", __FILE__)
-require 'rdoc'
+require_relative "lib/rdoc"
```

Comment:
Almost `require` in RDoc are with single-quote. The command below is for listing `require` with double-quote:

```bash
$ git grep '^require "'           
bin/console:require "bundler/setup"                        
bin/console:require "rdoc"                                 
bin/console:require "irb"                                  
lib/rdoc/ruby_lex.rb:require "e2mmap"                      
lib/rdoc/ruby_lex.rb:require "irb/slex"                    
lib/rdoc/ruby_lex.rb:require "stringio"                    
test/test_rdoc_parser_ruby.rb:require "\#{prefix}/a_library"
```

The `bin/console` is auto-generated code as you know, the `lib/rdoc/ruby_lex.rb` is forked source code from IRB and last one is dynamic string. So I think what you should use single-quote for this `require`.

Reply:
I chose double-quotes following the existing string literals in rdoc.gemspec. Do you want me to replace them all?

Reply:
I explained incompletely. The double quotes for `require` are stemmed from historical background. I just wanted to say about most `require` are used with single quote. But it's not important if I really think about it.